src/NServiceBus.Transport.SqlServer/SqlServerTransportInfrastructure.cs

```diff
@@ -342,6 +335,10 @@ class FakePromotableResourceManager : IEnlistmentNotification
         IDelayedMessageStore delayedMessageStore = new SendOnlyDelayedMessageStore();
         TableBasedQueueCache tableBasedQueueCache;
 
+        static string DtcErrorMessage = @"
+Note that distributed transactions are not available on Linux but the transport can still used by configuring a lower transaction mode via the `SqlServerTransport.TransportTransactionMode` property when configuring the endpoint.
```

Comment:
```suggestion
Distributed transactions are not available on Linux.  The other transaction modes can be used by setting the `SqlServerTransport.TransportTransactionMode` property when configuring the endpoint.
```
I don't know if this is better or not.  I wonder if "lower" is not the right word to use, but I can be overruled.   Also, do we typically use the markdown code block notation in error messages?